ansible/roles/telegraf/defaults/main.yml

```diff
@@ -0,0 +1,96 @@
+---
+# .. vim: foldmarker=[[[,]]]:foldmethod=marker
+
+# .. Copyright (C) 2021 Dr. Serge Victor <https://dr.sergevictor.eu/>
+# .. Copyright (C) 2020 DebOps <https://debops.org/>
+# .. SPDX-License-Identifier: GPL-3.0-or-later
+
+# .. _telegraf__ref_defaults:
+
+# debops.telegraf default variables
+# =================================
+
+# .. contents:: Sections
+#    :local:
+#
+# .. include:: ../../../../includes/global.rst
+
+
+# APT packages [[[
+# ------------
+
+# .. envvar:: telegraf__base_packages [[[
+#
+# List of the default APT packages to install Telegraf.
+telegraf__base_packages: [ 'telegraf' ]
+
+                                                                   # ]]]
+# .. envvar:: telegraf__packages [[[
+#
+# List of additional APT packages ta install with Telegraf.
+telegraf__packages: []
+
+                                                                   # ]]]
+                                                                   # ]]]
+
+# Telegraf configuration directory [[[
+# --------------------------------
+
+# The variable below defines the contents of the
+# :file:`/etc/telegraf/telegraf.d` configuration directory.
+# Definition of minimum one input and one output is mandatory.
+
+# .. envvar:: telegraf__configuration [[[
+#
+# All plugins you want to run on all hosts
+telegraf__configuration:
+
+  - name: 'input_influxdb_local'
+    config: |
+      [[inputs.influxdb_v2_listener]]
+        service_address = "127.0.0.1:38086"
+    state: 'present'
+
+  - name: 'output_discard'
+    config: |
+      [[outputs.discard]]
+    state: 'present'
```

Comment:
Are the above plugins useful in normal operations? Because with this setup, when users try to define their own configuration with `telegraf__configuration` variable in the inventory this configuration will be "masked" by Ansible merging the variables in the inventory. It might be better to move this configuration to `telegraf__default_configuration` variable and let users decide to explicitly remove these plugins via `telegraf__configuration` later on.